packages/field/src/Field.tsx

```diff
@@ -0,0 +1,229 @@
+import * as React from 'react';
+import { FieldExtensionSDK } from 'contentful-ui-extensions-sdk';
```

Comment:
I think this type is available via `field-editor-shared` and it would be to consume it that way. Ideally, `contentful-ui-extensions-sdk` shouldn't be a dependency of this package.